watertap/core/membrane_channel_base.py

```diff
@@ -1008,6 +1013,71 @@ def calculate_scaling_factors(self):
                 if iscale.get_scaling_factor(v) is None:
                     iscale.set_scaling_factor(v, 1)
 
+        # TODO why did this file have zero scaling factors for constraints
```

Comment:
@bknueven can probably give a more informative response, but the line of thinking was that scaling all variables should suffice in most cases without the need to scale constraints. However, we know that this isn't always the case, as we've experienced particularly when dealing with getting "complex" models to converge.